docs/actions/actions.md

```diff
@@ -44,30 +44,51 @@ if (kirby()->request()->is('POST')) {
 
 ## Custom Actions
 
-Custom actions are very similar to [custom guards](/guards/guards#custom-guards), too. Each action is a class in the `Uniform\Actions` namespace. To write your own actions you can either provide them through a PHP package and Composer or implement them in the traditional way as a Kirby plugin. The plugin may be a file `site/plugins/uniform-actions.php` that will be automatically loaded by Kirby.
+Custom actions are very similar to [custom guards](/guards/guards#custom-guards), too. Each action is a class in the `Uniform\Actions` namespace. To write your own actions you can either provide them through a PHP package and Composer or implement them in the traditional way as a Kirby plugin.
```

Comment:
You removed the last sentence here, but left it for the guard part. Might be better to keep it in sync

Reply:
Removed both